eisbuk-admin/src/store/selectors/__testData__/customers.ts

```diff
@@ -0,0 +1,24 @@
+import { LocalStore } from "@/types/store";
+
+import { createTestStore } from "@/__testUtils__/firestore";
+import { saul, walt, jian } from "@/__testData__/customers";
+
+/**
+ * A customers store input
+ */
+const customers: LocalStore["firestore"]["data"]["customers"] = {
+  [saul.secretKey]: saul,
+  [jian.secretKey]: jian,
+  [walt.secretKey]: walt,
```

Comment:
This will not change the behavior of tests/display, but customers are keyed by id, not secret key, so it would make sense to keep things consistent here as well.